packages/core/src/index.js

```diff
@@ -23,6 +23,7 @@ exports.registerAdditionalInstrumentations = function registerAdditionalInstrume
 
 exports.preInit = function preInit() {
   var preliminaryConfig = normalizeConfig();
+  exports.util.hasThePackageBeenInitializedTooLate();
```

Comment:
By coincidence the other changes in this commit uncovered that this call was missing from `preInit`. We need to run our initialized-too-late check in `preInit` too to avoid another class of false positives.